projects.json

```diff
@@ -488,6 +488,26 @@
     "discord": "https://discord.gg/fuelnetwork",
     "image": "fuelbridge"
   },
+  {
+    "isFeatured":true,
+    "isLiveMainnet": true,
+    "isLive": true,
+    "name": "Fuel Bridge Auto-Relayer",
+    "url": "https://app.fuel.network/bridge",
+    "tags": [
+      "Bridge",
+      "Fuel",
+      "Tooling"
+    ],
+    "hidden": true,
```

Comment:
for the fuel bridge auto-relayer we dont need the tags etc... it can be minimal info only as it won't show in projects list (follow mexc one)